fastly/connection.py

```diff
@@ -1,11 +1,14 @@
 """
 """
-
-import httplib
+import sys
+if sys.version_info[0] == 2:
+    import httplib
+else:
+    import http.client as httplib
```

Comment:
Instead of making it unclear what httplib refers to, do like you did for the urllib case and import HTTPConnection and HTTPSConnection.